### DISCUSSION TITLE
Stepper "data-on-step-click" not going to callback function

### DISCUSSION THREAD

**Discussion by Vamoshi** [olton/Metro-UI-CSS#1872] [Q&A]
Hello so I was introduced by my friend to Metro 4 and so far I am loving it. However, one very frustrating thing I've come across is that the "data-on-step-click" does not work in the stepper.

Here is my code snippet:
```
<div data-role="stepper"
     data-steps="6"
     data-view="cycle"
     data-clickable="true"
     data-click-step="true"
     data-on-step-click="onStepClick"
     data-on-step="onStep">
</div>
```

The onStepClick callback does not fire in this case, however I tried placing the function as the callback for "data-on-step", and the function is working fine there. I've been trying out different things the past few hours and I can't seem to make it work.

There's only this one line in the documentation talking about "data-on-step-click":

_onStepperCreate(element) | data-on-stepper-create | Fired when stepper created_

There's nothing else in the documentation I could find that was able to help me, not even an example for this data attribute, it just mentions it. Any assistance is greatly appreciated! Thanks!

This is the website where I'm reading the documentation from btw:
https://metroui.org.ua/stepper.html

**Comment by yyer2014** [comment]
https://metroui.org.ua/stepper.html contains a mistake: read and use data-step-click="true" instead of data-click-step="true" 
So check your attribute names, please.
